hw/ip/clkmgr/data/clkmgr_testplan.hjson

```diff
@@ -13,21 +13,90 @@
     {
       name: smoke
       desc: '''
-            Smoke test accessing a major datapath within the clkmgr.
+            Smoke test disabling peripheral and transactional clocks.
+
+            - Disables all peripheral clocks from their enabled reset state.
+            - Transactional clocks gating depends on whether they are idle.
+            - Initialize all units as busy (idle == '0).
+            - For each unit clear its hint bit, which has no effect until
+              the unit becomes idle.
+            - Set the unit's idle bit, which should disable the clock.
 
             **Stimulus**:
-            - TBD
+            - CSR writes to clk_enables and clk_hints.
+            - Setting idle_i clkmgs input.
 
             **Checks**:
-            - TBD
+            - SVA assertions for peripheral clocks enable and disable
+              properties.
+            - Transactional clocks check SVA properties as follows:
+              - If the hint enables it, the clock becomes active.
+              - If the hint disables it but the unit is busy, the clock remains
+                active.
+              - If the hint disables it and the unit is idle, the clock stops.
             '''
       milestone: V1
       tests: ["clkmgr_smoke"]
     }
     {
-      name: feature1
-      desc: '''Add more test entries here like above.'''
-      milestone: V1
+      name: peri_enables
+      desc: '''
+            Peripheral clocks are disabled unless pwr_i.ip_clk_en is set.
```

Comment:
nit: 
```suggestion
            Peripheral clocks are disabled unless `pwr_i.ip_clk_en` is set.
```
Consider putting all RTL specific items in backticks so that it gets highlighted in the doc.

Reply:
Done.